modules/channel-web/src/backend/socket.ts

```diff
@@ -12,6 +12,28 @@ export default async (bp: typeof sdk, db: Database) => {
   const config: any = {} // FIXME
   const { botName = 'Bot', botAvatarUrl = undefined } = config || {} // FIXME
 
+  async function incomingHandler(event: sdk.IO.Event, next: sdk.IO.MiddlewareNextCallback) {
+    if (event.channel !== 'web') {
+      return next()
+    }
+
+    if (event.type === 'postback' && event.target) {
+      const userId = event.target
+      const payload = bp.RealTimePayload.forVisitor(userId, 'webchat.postpack', event.payload)
+      bp.realtime.sendPayload(payload)
+    }
```

Comment:
If we're sending something to the user, then it's an outgoing event.  The postback is sent by the user using the API Endpoint, like other messages and events, i'd like to avoid adding another middleware if it's not absolutely necessary.